kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/modem/SupportedUsbModemInfo.java

```diff
@@ -53,9 +54,15 @@
             ModemTechnologyType.UMTS), Arrays.asList(new OptionModemDriver("19d2", "1476")), ""),
     SimTech_SIM7000("SIM7000", "1e0e", "9001", 5, 0, 3, 2, 3, 5000, 10000, Arrays.asList(ModemTechnologyType.LTE,
             ModemTechnologyType.GSM_GPRS), Arrays.asList(new OptionModemDriver("1e0e", "9001")), ""),
-    QUECTEL_EG25("EG25", "2c7c", "0125", 4, 0, 2, 3, -1, 5000, 10000, Arrays.asList(ModemTechnologyType.LTE,
+    QUECTEL_EG25("EG25", "2c7c", "0125", 4, 0, 3, 3, 1, 5000, 10000, Arrays.asList(ModemTechnologyType.LTE,
```

Comment:
Hi, are you sure that the Quectel EG 25 has gps support? Are the AT and data port the same?

Reply:
OK. The modem supports GNSS, but it's not supported yet in the code. I will revert this change.

Reply:
In fact EC25 and EG25 are identical from a host standpoint, so we will merge both into a single name: Ex_25